dfpl/options.py

```diff
@@ -534,20 +695,71 @@ def parseTrainGnn(parser: argparse.ArgumentParser) -> None:
     files_args = parser.add_argument_group("Files")
     model_args = parser.add_argument_group("Model arguments")
     training_args = parser.add_argument_group("Training Configuration")
+    uncertainty_args = parser.add_argument_group("Uncertainty Configuration")
+    uncertainty_args.add_argument(
+        "--uncertainty_method",
+        type=str,
+        metavar="STRING",
```

Comment:
Remove the `metavar` for all arguments that contain `choices`

```suggestion
```

Otherwise the choices are not shown on the CLI.

Reply:
Done!